CMakeLists.txt

```diff
@@ -132,14 +132,8 @@ include(SetupSphinx)
 include(CodeCoverageDetection)
 include(SpectreAddLibraries)
 
-option(RUN_TESTS_WITH_CHARMRUN
-  "Run test executables through charmrun.  Necessary on some clusters."
-  NO)
-if (RUN_TESTS_WITH_CHARMRUN)
-  set(SPECTRE_TEST_RUNNER ${CMAKE_BINARY_DIR}/bin/charmrun)
-else ()
-  set(SPECTRE_TEST_RUNNER "")
-endif ()
+set(SPECTRE_TEST_RUNNER "" CACHE STRING
```

Comment:
isn't this just `option(SPECTRE_TEST_RUNNER "description" "")`?

Reply:
I don't understand why, but the `option` form ignored the default value and this doesn't.